doc/release/RELEASE-NOTES.md

```diff
@@ -29,8 +40,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe what's new here. The changes that can impact user experience should be listed in this section.*
 
 ### Release instructions:
-* *Provide release instructions here.*
-
+* Update code.
+* Apply DB migrations up to the latest one.
```

Comment:
No need to run migrations for this release.

Reply:
thank you, removed